src/rpy_diffusive_flux.cc

```diff
@@ -142,18 +164,11 @@ void RPYDiffusiveFlux::setViscosity(double viscosity)
     viscosity_ = viscosity;
     }
     
-int RPYDiffusiveFlux::determineBufferShape(std::shared_ptr<State> state, const std::string& type)
+int RPYDiffusiveFlux::determineBufferShape(std::shared_ptr<State> state, const std::string&)
```

Comment:
I prefer to comment out the variable name like this if it is unused:
```suggestion
int RPYDiffusiveFlux::determineBufferShape(std::shared_ptr<State> state, const std::string& /* type */)
```